packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -149,8 +158,13 @@ export function getSpaces(sparkInstance, options = {}) {
                 const participants = decryptedSpace.participants.items;
                 const activityActors = activities.map((a) => a.actor);
 
-                dispatch(storeUsers([...participants, ...activityActors]));
-                dispatch(storeActivities(decryptedActivities));
+                if (shouldStoreUsers) {
```

Comment:
I'm excited for this...... but I'm thinking if anybody would even ask for that? like what is the point of storing activities and users?